### ISSUE TITLE
Since I haven't updated in a while; possible rewrite? [PUT ON HOLD]

### ISSUE THREAD

**Issue by Gerschel** [Gerschel/sd-web-ui-quickcss#36]
@Nacurutu 
I can't promise to devote time to it yet, I might have a couple of days between April 25th and May 1st, to attempt a parital rewrite.  
In the meantime, I might be able to squeeze in a few fixes. What seem to be the most pressing issues that need addressed sooner?  

Also. We can probably put together a checklist of things that should be in a rewrite.

**Comment by Nacurutu** [comment]
hi, welcome back.. no worries, your life and work first, then, the coding hobby...

I really don't know about the issues because I haven't updated the webUI, also, I'm full of work right now, didn't have the chance to check it, I was planning to install the webUI on a new folder on Sunday (a little of free time) to not mess with the one working "properly" and check for possible bugs/issues. I'm full of work and also I'm moving to a new place, this week has been a mess :p 

I hope I can help on Sunday.. but honestly, I cant promise.

**Comment by Gerschel** [comment]
Yep. It's not like we get paid, typically we get ripped off. No worries, we'll get there at some point.

**Comment by ParityError** [comment]
Yes. wb! I looked through the latest automatic1111 webui and it looks like the most recent version of gradio it uses is eventually going to support theming. There are loads of new vars showing up in the css that don't seem to be implemented yet. The quickcss extension still works (sorta), although there are some issues now with the color pickers, and of course they had to change the id and class names of things so it's a mess. 

The extension still provides maximum flexibility as far as customization is concerned; I hope we can keep it going when time permits.

**Comment by Gerschel** [comment]
We also have the option of custom classes through gradio now too.  https://github.com/gradio-app/gradio/issues/2962#event-8760312567

**Comment by ParityError** [comment]
I was able to get my "blueprint" .css to work properly with the latest commit of automatic1111 (since i couldn't stand the default gradio style). It took quite a bit of effort to track down all the changes in the latest gradio .css but I was able to resolve almost everything. If you want to provide it as a temporary fix for your extension let me know.

![Screenshot 2023-03-30 at 12 35 22 PM](https://user-images.githubusercontent.com/36368048/228945918-b40a2144-269c-4f14-b358-27639cb560dc.png)

**Comment by Piscabo** [comment]
> I was able to get my "blueprint" .css to work properly with the latest commit of automatic1111 (since i couldn't stand the default gradio style). It took quite a bit of effort to track down all the changes in the latest gradio .css but I was able to resolve almost everything. If you want to provide it as a temporary fix for your extension let me know.
> 
> ![Screenshot 2023-03-30 at 12 35 22 PM](https://user-images.githubusercontent.com/36368048/228945918-b40a2144-269c-4f14-b358-27639cb560dc.png)

Where can I pull this from? PLS.

**Comment by oakif** [comment]
@ParityError Hey! Also wanted to ask if you have a version of this somewhere. I just updated to the latest commit [22bcc7be](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/22bcc7be428c94e9408f589966c2040187245d81) in SD and it broke this extension from my previous update, which had been [3715ece0](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/3715ece0adce7bf7c5e9c5ab3710b2fdc3848f39).

**Comment by Gerschel** [comment]
This project is temporarily on hold until I get the other project working. This one would take longer to rework.

**Comment by oakif** [comment]
Appreciate the update! I currently have been using the below theme: 

https://github.com/canisminor1990/sd-webui-kitchen-theme

Hope someone finds it useful!
